tests/test_query.py

```diff
@@ -125,6 +125,42 @@ def test_get_user_requests(client):
     assert res.status_code == 401, res.text
 
 
+def test_get_active_user_requests(client):
+    fake_jwt = "1.2.3"
+
+    # create a request with no status
+    data = {
+        "policy_id": "test-draft-policy",
+        "resource_id": "draft_uniqid",
+        "resource_display_name": "My Draft Resource",
```

Comment:
you might want to explicitly make the status a draft status, instead of not specifying the status.
let's also create a request with a final status